src/main/scala/chisel3/internaltest/EnumHelpers.scala

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.internaltest
+
+import chisel3.experimental.EnumType
+
+/** Helper functions to allow for peeks and better debugging of ChiselEnums.
+  * This needs to be in a `chisel3` package in order to access the package private
+  * `factory` field of EnumType.
+  */
+object EnumHelpers {
+  def fromBits[T <: EnumType](tpe: T, bits: BigInt): T = {
+    val all = tpe.factory.all
+    all.find(_.litValue == bits).get.asInstanceOf[T]
```

Comment:
Why is the .asInstanceOf[T] cast required here? I know it's done pretty much all over the place, but I'm still not sure why